packages/core/src/routes/admin-user/mfa-verifications.ts

```diff
@@ -0,0 +1,152 @@
+import { MfaFactor, userMfaVerificationResponseGuard } from '@logto/schemas';
+import { getUserDisplayName } from '@logto/shared';
+import { authenticator } from 'otplib';
+import qrcode from 'qrcode';
+import { object, string, z } from 'zod';
+
+import RequestError from '#src/errors/RequestError/index.js';
+import koaGuard from '#src/middleware/koa-guard.js';
+import assertThat from '#src/utils/assert-that.js';
+import { transpileUserMfaVerifications } from '#src/utils/user.js';
+
+import { generateBackupCodes } from '../interaction/utils/backup-code-validation.js';
+import { generateTotpSecret } from '../interaction/utils/totp-validation.js';
+import type { AuthedRouter, RouterInitArgs } from '../types.js';
+
+export default function adminUserMfaVerificationsRoutes<T extends AuthedRouter>(
+  ...args: RouterInitArgs<T>
+) {
+  const [
+    router,
+    {
+      queries,
+      libraries: {
+        users: { addUserMfaVerification },
+      },
+    },
+  ] = args;
+  const {
+    users: { findUserById, updateUserById },
+  } = queries;
+
+  router.get(
+    '/users/:userId/mfa-verifications',
```

Comment:
@simeng-li new apis should have swagger descriptions. let's add together to cover all apis

Reply:
we have a general issue for this

Reply:
For MFA related endpoints: LOG-7325